index.html

```diff
@@ -12,9 +12,119 @@
     <title>Проект</title>
   </head>
   <body>
-    <div class="container">
-      <h1 style="text-align: center">ПУСТОЙ ПРОЕКТ</h1>
-    </div>
+    <header class="header">
+      <div class="header container">
+        <div class="header__left">
```

Comment:
2 блока header очень странно - поправить
header - оставить только цвет и создать
header__inner -  и у него настроить все остальное, флексы и т.д

```suggestion
    <header class="header">
      <div class="header__inner container">
        <div class="header__left">
```